draft-ietf-jsonpath-base.md

```diff
@@ -1528,9 +1532,6 @@ A function expression is well-typed if all of the following are true:
 * If it occurs directly as a `comparable` in a comparison, the
   function is declared to have a result type of `ValueType`, or
   (conversion applies) `NodesType`.
-* Otherwise, it is occurring as an argument to a further function
```

Comment:
I still like this text, as it rounds out the cases in the outer list.

Reply:
I had removed it because the changes I had made below it made it moot.  Without those other changes, I think this is still necessary.

Reply:
Ok, I'll put the bullet back with a minor rewording.